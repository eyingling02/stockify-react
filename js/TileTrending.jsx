const React = require('react')
const TileTrending = (props) => (
    <div className='outer'>
        <div className='buy'>
            <a href='javascript:;'>UP</a>
        </div>
        <div className='inner'>
            <a href='livingroom.html' target='_blank'>
                <h2>{props.Symbol}</h2>
                <h2>{props.ChangePercent}%</h2>
                <h3>{props.CompanyName} days</h3>
            </a>
        </div>
        <div className='sell'>
            <a href='javascript:;'>DOWN</a>
        </div>
    </div>
)

const {string} = React.PropTypes

module.exports = TileTrending
