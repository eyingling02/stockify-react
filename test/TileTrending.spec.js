import React from 'react'
import TileTrending from '../js/TileTrending.jsx'
import { shallow } from 'enzyme'
import { shallowToJson } from 'enzyme-to-json'

test('TileTrending snapshot test', () => {
  const component = shallow(<TileTrending />)
  const tree = shallowToJson(component)
  expect(tree).toMatchSnapshot()
})
