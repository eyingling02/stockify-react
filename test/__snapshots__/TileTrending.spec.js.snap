exports[`test TileTrending snapshot test 1`] = `
<div
  className="outer">
  <div
    className="buy">
    <a
      href="javascript:;">
      UP
    </a>
  </div>
  <div
    className="inner">
    <a
      href="livingroom.html"
      target="_blank">
      <h2 />
      <h2>
        %
      </h2>
      <h3>
         days
      </h3>
    </a>
  </div>
  <div
    className="sell">
    <a
      href="javascript:;">
      DOWN
    </a>
  </div>
</div>
`;
